test/compile-error-test/CMakeLists.txt

```diff
@@ -1,77 +1,128 @@
 # Test if compile errors are produced where necessary.
 
 cmake_minimum_required(VERSION 3.1...3.18)
+project(compile-error-test)
```

Comment:
What is this for?

Reply:
It's needed for the same reason as here - https://github.com/fmtlib/fmt/pull/2648#discussion_r772076387.
Also, other tests have this `project()` call already:
https://github.com/fmtlib/fmt/blob/c5aafd8f9003c5f0ac1d9bc00d566fefe60996d7/test/add-subdirectory-test/CMakeLists.txt#L3
https://github.com/fmtlib/fmt/blob/c5aafd8f9003c5f0ac1d9bc00d566fefe60996d7/test/find-package-test/CMakeLists.txt#L3

Reply:
OK, let's keep project then but what's the problem with `CXX`? It's usually a good idea to have `CXX` because it reduces the number of checks.

Reply:
But there is no problem with `CXX` 🤔
Oh, you mean my _marked as outdated_ addition - it's about this GitHub discussion thread only. Just to indicate that I didn't remove this line but changed it.